opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -30,36 +30,68 @@
 
 import os
 import sys
+from collections import Iterable
 import opentimelineio as otio
 
 lib_path = os.environ.get("OTIO_AAF_PYTHON_LIB")
 if lib_path and lib_path not in sys.path:
-    sys.path += [lib_path]
+    sys.path.insert(0, lib_path)
```

Comment:
If we're moving to pyaaf2 with pure python, then we should put the pip library in the setup.py, and trust that it will be installed. We should no longer need lines ~35-39 at all.